Source/WebKit/WPEPlatform/wpe/wayland/WPEWaylandCursor.cpp

```diff
@@ -92,6 +94,8 @@ void WaylandCursor::setFromBuffer(struct wl_buffer* buffer, uint32_t width, uint
 
 void WaylandCursor::update() const
 {
+    if (!m_theme)
+        return;
```

Comment:
This still breaks custom cursors. I think we need another member variable to set whether the cursor changed to return early here.